shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -53,4 +53,8 @@ FLUTTER_EXPORT
     NS_DESIGNATED_INITIALIZER;
 - (nonnull instancetype)initWithCoder:(nonnull NSCoder*)nibNameOrNil NS_DESIGNATED_INITIALIZER;
 
+- (void)handleMethodCall:(nonnull FlutterMethodCall*)call result:(nonnull FlutterResult)result;
+
+- (nonnull NSPasteboard*)getPasteboard;
```

Comment:
This is non idiomatic objective-c, you don't use "getX" in objective c, you use properties, ex 
```objc 
@property(nonatomic, readonly, nonnull) NSPasteBoard* _pasteboard;
```

Reply:
Thanks for the tip!  This way seems cleaner and is just as mockable.